widget/embedded/src/components/SwapDetails/SwapDetails.Placeholder.tsx

```diff
@@ -23,8 +23,8 @@ import {
 } from './SwapDetails.styles';
 
 export function SwapDetailsPlaceholder(props: SwapDetailsPlaceholderPropTypes) {
-  const { requestId, showSkeleton } = props;
-
+  const { requestId } = props;
+  const showSkeleton = true;
```

Comment:
If this variable isn't need anymore, please remove it. and also please remove the type.